lib/rails_stdout_logging/rails.rb

```diff
@@ -1,6 +1,16 @@
 module RailsStdoutLogging
   class StdoutLogger < ::Logger
     include ::LoggerSilence if defined?(::LoggerSilence)
+
+    def initialize(*args)
+      super
+      after_initialize if respond_to? :after_initialize
+    end
+
+    def add(severity, message = nil, progname = nil, &block)
+      return true if @logdev.nil? || (severity || UNKNOWN) < level
```

Comment:
What is this unknown costant coming from? Can we use the full thing i.e. `::Logger::Unknown`